.github/workflows/approver.yml

```diff
@@ -4,6 +4,10 @@ on:
   issue_comment:                                     
     types: [created, edited]
 
+env:
+  # sintax: ("germa89" "author2")
```

Comment:
Hi @germa89. We love this action and using in PyVista project too. I have one question in https://github.com/pyvista/pyvista/pull/5314. Is there any reason why we cannot use comma(,) between authors? Thanks for your great job.

Reply:
Hi @tkoyama010 ! I am not sure why. I guess the GitHub run parse that env var again when passing it to the shell script that later execute for the job. But I don't know. I didn't really face the issue I think. Sorry for not being more helpful!

Reply:
Never mind. The author's comments are very valuable. I will test it when I get a chance.

Reply:
Sure. But I think it will be a little later to test it.

Reply:
See https://github.com/pyvista/pyvista/pull/5327 .